src/mmw/js/src/analyze/templates/error.html

```diff
@@ -0,0 +1,8 @@
+<div class="analyze-message-inner">
```

Comment:
Looks like error.html and loading.html are no longer used, and maybe there should be a message.html to replace them?